core/gui/src/app/dashboard/component/user-dashboard-test-fixtures.ts

```diff
@@ -40,7 +39,7 @@ export const testWorkflowContent = (operatorTypes: string[]): WorkflowContent =>
   commentBoxes: [],
   links: [],
   operatorPositions: {},
-  settings: { dataTransferBatchSize: environment.defaultDataTransferBatchSize },
+  settings: { dataTransferBatchSize: 400 },
```

Comment:
Another copy of the default value here. The default value of batch size can change from 400 at any time.

Reply:
This class is used for providing Mock values for gui's unit testing. That's why I provide a hard code value here.